build/release-images.sh

```diff
@@ -31,8 +31,11 @@ if [[ "${KUBE_BUILD_CONFORMANCE}" =~ [yY] ]]; then
 fi
 
 # TODO(dims): Remove this when we get rid of hyperkube image
-CMD_TARGETS="${CMD_TARGETS} cmd/kubelet cmd/kubectl"
+if [[ "${KUBE_BUILD_HYPERKUBE}" =~ [yY] ]]; then
+    CMD_TARGETS="${CMD_TARGETS} cmd/kubelet cmd/kubectl"
+fi
 
+CMD_TARGETS=$(echo "${CMD_TARGETS}" | tr ' ' '\n' | sort | uniq | tr '\n' ' ')
```

Comment:
Can you please move this line into the `if` condition? Since we only run into this for `KUBE_BUILD_HYPERKUBE` is set to `y` or `Y`?

Reply:
@dims there is also the conformance image targets ...?